.circleci/config.yml

```diff
@@ -0,0 +1,69 @@
+version: 2
+jobs:
+  build:
+    docker:
+      # Specify the version you desire here
+      - image: circleci/php:5.6-node
+      - image: circleci/mysql:5.7
+        environment:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+          MYSQL_USER: root
+          MYSQL_ROOT_PASSWORD: ''
+          MYSQL_DATABASE: jitterbug_testing
+
+    working_directory: ~/jitterbug
+
+    steps:
+      - checkout
+      - run:
+          name: install dockerize
+          command: wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && sudo tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+          environment:
+            DOCKERIZE_VERSION: v0.3.0
+      - run:
```

Comment:
The tests might run a bit faster is this was moved down right above the Migrate DB command

Reply:
ok i'll try it

Reply:
ohhh i think the reason i had it up here was because in one of the examples they had this wait command before they installed the mysql client extension, but that order probably isn't necessary

Reply:
Yeah, installing the extension shouldn't cause a connection attempt.